sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_prebuilts.py

```diff
@@ -182,6 +184,9 @@ def test_invoice_jpg(self, client, **kwargs):
         poller = client.begin_analyze_document("prebuilt-invoice", invoice)
 
         result = poller.result()
+        d = result.to_dict()
```

Comment:
invoices have nested items right? just want to make sure we're converting those to_dict() as well.

Reply:
I'm using invoices to get the CurrencyValue test, but receipt which is the other test does have the nested items.